lib/logidze/versioned_association.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+module Logidze
+  module VersionedAssociation
+    def load_target
+      target = super
+
+      return target if inversed
```

Comment:
I made a spec for inversed association. Correct me, if I wrong: in case of `belongs_to` inversed `has_many` is not populated anyway, so we don't care about it, right? E.g. in pseudo-code
```
user = article.user
user.articles.first == article # > false
```